hollow-ui-tools/src/main/java/com/netflix/hollow/ui/HollowUIRouter.java

```diff
@@ -16,19 +16,34 @@
  */
 package com.netflix.hollow.ui;
 
+import java.io.IOException;
 import java.io.InputStream;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 import org.apache.commons.io.IOUtils;
 import org.apache.velocity.app.VelocityEngine;
 import org.apache.velocity.runtime.RuntimeConstants;
 import org.apache.velocity.runtime.resource.loader.ClasspathResourceLoader;
 
-public abstract class HollowUIRouter {
+public abstract class HollowUIRouter extends HttpServlet {
 
     protected final String baseUrlPath;
     protected final VelocityEngine velocityEngine;
 
+    public void init() throws ServletException
+    {
+    }
+    public void doGet(HttpServletRequest request,
```

Comment:
Would be good to add @Override annotation where applicable

Reply:
done